types/splitio.d.ts

```diff
@@ -1228,6 +1232,19 @@ declare namespace SplitIO {
    * User consent status.
    */
   type ConsentStatus = 'GRANTED' | 'DECLINED' | 'UNKNOWN';
+  /**
+   * Fallback treatment can be either a string (Treatment) or an object with treatment and config (TreatmentWithConfig).
+   */
+  type FallbackTreatment = TreatmentWithConfig;
```

Comment:
Since `FallbackTreatment` is just an alias here, I would remove this type and use `TreatmentWithConfig` directly instead.